transitmatters/t-performance-dash
# Add Layouts

## Motivation

We want to be able to customize layouts depending on page. Specifically, the landing page does not need controls and a header.

## Changes
Add secondary layout component to _app.tsx which applies layout for each page.

## Testing Instructions
Go to root URL `/` and see that title and controls are gone. Other pages should remain the same.
